<?php

namespace Container2YcVERS;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getLocationIsNotSubLocationValidatorService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private 'Ibexa\AdminUi\Validator\Constraints\LocationIsNotSubLocationValidator' shared autowired service.
     *
     * @return \Ibexa\AdminUi\Validator\Constraints\LocationIsNotSubLocationValidator
     */
    public static function do($container, $lazyLoad = true)
    {
        include_once \dirname(__DIR__, 4).'/vendor/symfony/validator/ConstraintValidatorInterface.php';
        include_once \dirname(__DIR__, 4).'/vendor/symfony/validator/ConstraintValidator.php';
        include_once \dirname(__DIR__, 4).'/vendor/symfony/validator/Constraints/AbstractComparisonValidator.php';
        include_once \dirname(__DIR__, 4).'/vendor/ibexa/admin-ui/src/lib/Validator/Constraints/LocationIsNotSubLocationValidator.php';

        return $container->privates['Ibexa\\AdminUi\\Validator\\Constraints\\LocationIsNotSubLocationValidator'] = new \Ibexa\AdminUi\Validator\Constraints\LocationIsNotSubLocationValidator(($container->privates['property_accessor'] ?? $container->getPropertyAccessorService()));
    }
}
